HttpResponseImpl.java

```diff
@@ -0,0 +1,43 @@
+import java.io.OutputStream;
+import java.util.Map;
+
+public class HttpResponseImpl implements HttpResponse{
+    private String version;
+    private int StatusCode;
+    private String StatusMessage;
+    private Map<String,String> header;
+    private OutputStream body;
+
+
+    
+    public void HttpResponseImpl(OutputStream body) {
+        this.body = body;
+
+    }
+ 
+
+    public void setVersion(String version) {
+        this.version = version;
+    }
+
+
+    public void setStatusCode(int StatusCode) {
+        this.StatusCode = StatusCode;
+
+    }
+
+    public void setStatusMessage(String StatusMessage) {
+        this.StatusMessage = StatusMessage;
```

Comment:
Igual toca cambiar esto tambien.
```suggestion
    public void setStatusCode(int statusCode) {
        this.statusCode = statusCode;

    }

    public void setStatusMessage(String statusMessage) {
        this.statusMessage = statusMessage;
```